### DISCUSSION TITLE
Keypoint and Keyframe

### DISCUSSION THREAD

**Discussion by TRINHHOANGANH** [stella-cv/stella_vslam#475] [Q&A] [answered]
HI @All
I want to create an application so that it adds the keypoint of previous keyframes to the next image.
And, I choose the keypoint to load the next image, such as the image below
![Screenshot from 2023-02-10 10-51-04](https://user-images.githubusercontent.com/83366198/219581533-08dbb4cc-232e-4294-ba12-bee29b86cacb.png)

You can give me solution or how to get keypoint and add keypoint to frame?

**Comment by ymd-stella** [comment, marked as answer]
Why did you create an Issue when there are Discussions here?
It is mentioned in the Issue template. Yet you posted the question as a bug report.

Did you not read the following text?

![image](https://user-images.githubusercontent.com/7959916/219640710-b58e7ca7-7aaa-44be-8f03-b038910ae3ec.png)

It is unlikely that someone would create an Issue without reading the text in the Issue template, so it may not have shown up on your screen. Tell me whether it was displayed or not.

**Reply by ymd-stella** [reply]
If you continue to behave in this way, I will have to block you.

**Reply by ymd-stella** [reply]
The exact same question is posted at https://github.com/stella-cv/stella_vslam/discussions/472.

**Reply by TRINHHOANGANH** [reply]
Hi @ymd-stella 
I'm sorry. I didn't see it show up but now it's showing on my screen.
